fuzzing/README.md

```diff
@@ -0,0 +1,35 @@
+# Fuzzing
+
+Automated fuzzing tests for RIOT network applications.
+
+## Setup
+
+The following additional dependencies are required:
+
+* [afl][afl homepage]
+* [libasan][sanitizers github] (optional but recommended)
+
+## Invocation
+
+Before fuzzing an application it needs to be compiled, to ease detection
+of unwanted behaviour (e.g. out-of-bounds buffer accesses), compiling
+with `all-asan` is highly recommended. For example:
+
+	make -C fuzzing/<application> all-asan
+
+Afterwards invoke afl using:
+
+	make -C fuzzing/<application> fuzz
```

Comment:
It would be great if you could add a description of how to build a fuzzable application.
How do you get the data in? What should the application do (just read a packet and terminate or run a loop, etc?)

Reply:
Added a section entitled `Writing a fuzzing application` where I briefly explain the architecture of these applications and which operations need to be performed by them.